dashboards/google-application-integration/integration-executions.json

```diff
@@ -1,501 +1,417 @@
-                    "crossSeriesReducer": "REDUCE_SUM",
-                    "groupByFields": [],
-                    "perSeriesAligner": "ALIGN_SUM"
-                  },
-                  "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"is_retry\"=\"true\""
-                }
+        "width": 4,
+        "xPos": 4,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "blankView": {},
+            "thresholds": [],
+            "timeSeriesQuery": {
+              "apiSource": "DEFAULT_CLOUD",
+              "outputFullDuration": true,
+              "timeSeriesFilter": {
+                "aggregation": {
+                  "alignmentPeriod": "60s",
+                  "crossSeriesReducer": "REDUCE_SUM",
+                  "groupByFields": [],
+                  "perSeriesAligner": "ALIGN_SUM"
+                },
+                "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"status\"=\"CANCELLED\""
               }
-            },
-            "title": "Total Integrations Retried"
+            }
           },
-          "width": 3,
-          "xPos": 9,
-          "yPos": 0
+          "title": "Total canceled executions"
         },
-        {
-          "height": 4,
-          "widget": {
-            "title": "Integration executions with at least one error (per min)",
-            "xyChart": {
-              "chartOptions": {
-                "mode": "COLOR"
-              },
-              "dataSets": [
-                {
-                  "minAlignmentPeriod": "60s",
-                  "plotType": "LINE",
-                  "targetAxis": "Y1",
-                  "timeSeriesQuery": {
-                    "apiSource": "DEFAULT_CLOUD",
-                    "timeSeriesFilter": {
-                      "aggregation": {
-                        "alignmentPeriod": "60s",
-                        "crossSeriesReducer": "REDUCE_SUM",
-                        "groupByFields": [],
-                        "perSeriesAligner": "ALIGN_SUM"
-                      },
-                      "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"error_enum\"!=\"ERROR_ENUM_UNSPECIFIED\"",
-                      "secondaryAggregation": {
-                        "alignmentPeriod": "60s",
-                        "crossSeriesReducer": "REDUCE_NONE",
-                        "perSeriesAligner": "ALIGN_NONE"
-                      }
-                    }
+        "width": 4,
+        "xPos": 8,
+        "yPos": 0
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "Error (atleast one) containing executions per min",
```

Comment:
atleast -> at least

Reply:
Reading this again, I think the original phrasing was clearer? Executions with at least one error per min

Reply:
Changed it.